libraries/src/CMS/Application/ApiApplication.php

```diff
@@ -267,4 +266,19 @@ public function dispatch($component = null)
 		PluginHelper::importPlugin('system');
 		$this->triggerEvent('onAfterDispatch');
 	}
+
+	/**
+	 * Returns the application \JMenu object.
+	 *
+	 * @param   string  $name     The name of the application/client.
+	 * @param   array   $options  An optional associative array of configuration settings.
+	 *
+	 * @return  AbstractMenu|null
+	 *
+	 * @since   3.2
+	 */
+	public function getMenu($name = null, $options = array())
+	{
+		return null;
```

Comment:
Not sure what to do about this. This was a hack for working around a menu item being required at: https://github.com/joomla-projects/gsoc17_webservices/blob/4.0-dev/libraries/src/CMS/Controller/Controller.php#L750